armi/reactor/reactors.py

```diff
@@ -319,8 +321,13 @@ def lib(self, value):
     def isFullCore(self):
         """Return True if reactor is full core, otherwise False."""
         # Avoid using `not core.isFullCore` to check if third core geometry
-        # use `core.symmetry == geometry.THIRD_CORE + geometry.PERIODIC`
-        return self.symmetry == geometry.FULL_CORE
+        # use `core.symmetry.domain == geometry.DomainType.THIRD_CORE
+        return self.symmetry.domain == geometry.DomainType.FULL_CORE
+
+    @property
+    def isThirdCore(self):
+        """Return True if reactor is third core, otherwise False."""
+        return self.symmetry.domain == geometry.DomainType.THIRD_CORE
```

Comment:
Was there already an `isThirdCore()` on `Core`? This probably shouldnt be the responsibility of the Core at all, especially if it's this specific to 1/3 core. If someone needs to know if a Core is 1/3, it can peek at the `symmetry` itself.

Reply:
I guess not. I implemented it because there is a comment in `isFullCore` that says:

"avoid using `not core.isFullCore` to check for third core geometry, use `core.symmetry == geometry.THIRD_CORE + geometry.PERIODIC` instead."

I assumed checking third core geometry would be a useful function based on the comment, but I guess it's not actually used anywhere. It's something I put in early thinking I would use it later and then forgot about it.